### PR TITLE
Refresh Admin UI docs

### PR DESCRIPTION
**Pull request by mechanomi**
this PR refreshes the Admin UI docs to match the latest version and brings the formatting and style up-to-date with current standards

### CODE REVIEW

File: docs/console.rst [hunk 1]
```diff
@@ -1,62 +1,74 @@
...
+
 .. _sql-console:
 
 ===========
 SQL console
 ===========
 
-The CrateDB admin UI comes with an SQL console so that you can execute queries
-against your cluster directly from your web browser.
+The :ref:`CrateDB Admin UI <index>` comes with an *SQL console* so that you can
```
Comment (infoverload) [inline docs/console.rst:10]: Have we decided on whether it should be "a SQL" or "an SQL" ?

File: docs/console.rst [hunk 2]
```diff
@@ -1,62 +1,74 @@
...
+
 Screenshots
 ===========
 
 When you first load the console, it will look like this:
 
-.. image:: console-default.png
+.. image:: _assets/img/console-default.png
 
 And here's what the console looks like after executing a query:
```
Comment (infoverload) [inline docs/console.rst:28]: here is

**Review by infoverload — COMMENTED** [review]
All of the screenshots are cut off.  I understand that we can't always include the whole screen, but maybe it will look better to at least include the whole left hand navigation bar in the screenshots?

**Comment by infoverload** [comment]
@mechanomi wondering what your thoughts are on my screenshot comments?

**Comment by mechanomi** [comment]
@infoverload yup, I agree. just need to run through and generate them again

File: docs/console.rst (hunk 1 above)
Comment (mechanomi) [reply docs/console.rst:10]: I am not sure we came to a conclusion. however, based on https://english.stackexchange.com/a/5266, I currently believe that "an SQL" is the best choice

**Comment by mechanomi** [comment]
I have addressed the comments with a fixup

unfortunately, using the 4.3.1 release, I am unable to produce a new screenshot for the monitoring overview screen because of https://github.com/crate/crate/pull/10759

the fix for this is in the nightly release, but I am running into a second issue with that: the `bin/crate` script itself errors because of a Java issue. I am discussing this internally to see if I can figure out how to resolve the issue. otherwise, I will have to wait until 4.3.2 is released

**Comment by mechanomi** [comment]
still blocked on crate 4.3.2 release

**Comment by mechanomi** [comment]
@infoverload heya! I pushed the fixes you requested. can you take a look please? thx!!

**Review by matthijskrul — APPROVED** [review]

**Pull request merged**